apprunner.yaml

```
@@ -0,0 +1,21 @@
+run:
+  runtime-version: 1.21
+  command: ./main
+  network:
+    port: 8080
+  env:
+    - name: PORT
+      value: "8080"
+    - name: AWS_REGION
+      value: "ap-northeast-1"
+    - name: DB_SECRET_ARN
+      value: "arn:aws:secretsmanager:ap-northeast-1:ACCOUNT_ID:secret:PROJECT_NAME/database"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)